### PR TITLE
Small tweaks to language and quiet the markdown linter.

### PR DESCRIPTION
**Pull request by freeformz**
This is a re-open of #150.

### CODE REVIEW

**Comment by mars** [comment]
Hi @freeformz, did you build the docs locally and verify that they still render correctly? (Hashicorp's website markdown doesn't always parse & render as expected.)

**Comment by mars** [comment]
Also, what Markdown linter are you using?

File: website/docs/index.html.markdown [hunk 1]
```diff
@@ -3,28 +3,31 @@ layout: "heroku"
 page_title: "Provider: Heroku"
 sidebar_current: "docs-heroku-index"
 description: |-
-  The Heroku provider is used to interact with the resources supported by the Heroku Platform API.
+The Heroku provider is used to interact with the resources provided by the
```
Comment (mars) [inline website/docs/index.html.markdown]: This change breaks the layout:

Screen Shot 2019-03-28 at 10 10 25 AM

It's YAML front matter, not Markdown, so I guess the linter is misleading you here.

Comment (freeformz) [reply website/docs/index.html.markdown]: Fixed

**Comment by freeformz** [comment]
@mars I did a `make website` and looked the docs over. I am using the linter that is in visual studio code `markdownlint` extension: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

**Pull request merged**